setup.py

```diff
@@ -56,7 +56,7 @@
             "phiflow",
             "flyvision @ https://github.com/TuragaLab/flyvis/archive/refs/heads/main.zip",
         ],
-        "tests": ["networkx"],
+        "tests": ["networkx", "h5py"],
```

Comment:
I added h5py to the "tests" optional dependency list. It's imported in the arena module under examples/olfaction and used for the plume task.